### PR TITLE
Weather app project

### PR DESCRIPTION
**Pull request by nriqu322**
This weather app project was built with Javascript using webpack.

In this project you are able to :
- Enter a city.
- Check the weather of the day.
- Check a 5 days forecast.
- Change the temperature unit between °C and °F.
- See a funny gift.
- Use the OpenWeatherMap API.
- Use the Giphy API.

### CODE REVIEW

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @nriqu322 
Great job so far :clap: 
Your application looks very nice :smile:
We only need to make some changes before moving on.

- [x] Clicking on the toggle control to change between Celsius and Fahrenheit should update the temperature for the selected scale. Right now your app only changes if the user search for a new location.
Screenshot from 2020-07-28 09-34-07

- [x] The application should change its appearance based on the current forecast. This can be a different background related to the current weather, or you can simply display the icon forecast (the one that you are already using for future forecasts) to the current forecast.
Screenshot from 2020-07-28 09-37-51

- [x] It seems the day 5 forecast has a little bug, the weekday it's not being displayed for some reason. This is happening both on the live version and when I run your app locally.
Screenshot from 2020-07-28 09-42-26

- [x] Nice that you implemented a `build` script to the package.json file, but it's a best practice to also inform in which mode Webpack should run ( development or production). Since the purpose of the watch script it for the development stage, you also should also inform that Webpack should run in development mode. Your script will look like something like this:
`  "watch": "webpack --mode development --watch",`
`  "build": "webpack --mode production"`

- [x] You should also add to your README file, instructions on how to use your app, describing the scripts from the package.json file.

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**

File: README.md [hunk 1]
```diff
@@ -6,10 +6,15 @@ This is a weather app where you can enter a city to check the weather in °C or
...
+- Check the weather of cities.
+- Actual temperature.
+- Min and Max temperature during the day.
+- A description of the weather.
+- Check a 5 days forecast.
+- See a funny gift.
 
 ## Project specifications
-* [The Odin Project: ToDo List](https://www.theodinproject.com/courses/javascript/lessons/todo-list)
+* [The Odin Project: ToDo List](https://www.theodinproject.com/courses/javascript/lessons/weather-app)
```
Comment (Torres-ssf) [inline README.md]: - [x] This link name is related to another project

**Review by Torres-ssf — CHANGES_REQUESTED** [review]

**Comment by nriqu322** [comment]
Made all changes:

- Background image changes with the weather description
- Bug in forecast fixed
- Able to change between °C and °F
- README file updated.

**Review by shshamim63 — APPROVED** [review]
## :tada: Approved
___
Hi @nriqu322 :wave: 
Well done. All the issues are fixed mentioned by the previous reviewer :clap: . Just found an issue and you can resolve it easily. And it is not mandatory to change, but good to add that.
image
It does not matter I get the city or not, the page always informs me that, the city not found. You should not provide vague information. That is it.
___
##### This :+1: is from me, you can merge it.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain

**Pull request merged**